release-process.md

```diff
@@ -35,6 +35,34 @@ If you are a new Release Manager, you can read up on the process from the follow
 - gpg for signing https://www.apache.org/dev/openpgp.html
 - svn https://www.apache.org/dev/version-control.html#https-svn
 
+<h3>Preparing gpg key</h3>
+
+You can skip this section if you have been a release manager before.
```

Comment:
"if you have been a release manager before" => "if you've already uploaded your key". No need to be a release manager before :).